site/pages/_app.tsx

```diff
@@ -0,0 +1,7 @@
+import 'bootstrap/dist/css/bootstrap-reboot.min.css'
```

Comment:
This being an Emotion-based site, we should avoid using external stylesheets - this won't come in the SSRed output so we might end up with FOUC etc

Reply:
Won't Next add a `<link>` tag to the `<head>` when rendering on the server? Thus preventing FOUC?

Regardless, I can avoid dependending on Reboot if you prefer. I just like not having to worry about fonts, heading margins, .etc.

Reply:
> Won't Next add a <link> tag to the <head> when rendering on the server? Thus preventing FOUC?

Ah, yes - you are right, there wouldnt be FOUC with link here.

> Regardless, I can avoid dependending on Reboot if you prefer. I just like not having to worry about fonts, heading margins, .etc.

Ye, i would prefer removing this dependency. If it’s just about CSS reset then there are plenty that we could just copy over here. The one in Chakra UI was sufficient for me in my latest projects.